masterlist.yaml

```diff
@@ -17921,6 +18202,7 @@ plugins:
     after:
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'ELE_Legendary_Lite.esp'
+    global_priority: 77
```

Comment:
What were your thoughts on picking this specific `global_priority`?

(Also, https://loot.github.io/docs/contributing/Global-Priorities.html really ought to get updated. :/)